assets/js/responsive_tables.js

```diff
@@ -0,0 +1,20 @@
+// set data-title attribute on table cells
+function responsiveTables() {
+  const tables = document.getElementById("main-content-wrapper").getElementsByTagName("table")
+  for (table of tables) {
```

Comment:
The `for` statements here should have a `const` to define the variable, for example `for (const table of tables) {`